wikipedia-dark.user.css

```diff
@@ -1,7 +1,7 @@
 /* ==UserStyle==
 @name         Wikipedia Dark
 @namespace    StylishThemes
-@version      3.5.10
+@version      3.5.11
```

Comment:
Please don not include version bumps, this is strictly 100% for internal use.

Reply:
k